db/migrate/20240424084932_create_message_submission_requests.rb

```diff
@@ -0,0 +1,10 @@
+class CreateMessageSubmissionRequests < ActiveRecord::Migration[7.1]
+  def change
+    create_table :message_submission_requests do |t|
+      t.belongs_to :box, null: false, foreign_key: true
+      t.string :request_url
+      t.integer :response_status
+      t.timestamps
```

Comment:
Uvazujem, ci si sem nechceme ulozit aj message UUID. Neviem ale ci potrebujeme, jedine ak by klient chcel od nas, nech dodame podklady, kde na akych sprav sa mu to naratalo pri fakturacii.